components/widgets/menu/menu_items/menu_cloud_trial.test.tsx

```diff
@@ -37,28 +37,6 @@ describe('components/widgets/menu/menu_items/menu_cloud_trial', () => {
         expect(wrapper.find('UpgradeLink').exists()).toEqual(true);
     });
 
-    test('should NOT render when NOT on cloud license and NOT during free trial period', () => {
```

Comment:
Is this behavior related to the user limit? It seems that it's testing if the Upgrade to trial link is available or not? We may want to keep this test